src/main/java/uk/gov/hmcts/reform/civil/service/documentmanagement/ClaimFormService.java

```diff
@@ -25,11 +27,7 @@ public CaseDocument uploadSealedDocument(
     @Autowired
     private final SealedClaimFormGeneratorForSpec sealedClaimFormGeneratorForSpec;
 
-    public byte[] downloadSealedDocument(String authorisation, CaseDocument caseDocument) {
-        return sealedClaimFormGeneratorForSpec.downloadDocument(caseDocument, authorisation);
-    }
-
-    public byte[] downloadSealedDocument(CaseDocument caseDocument) {
-        return sealedClaimFormGeneratorForSpec.downloadDocument(caseDocument);
+    public ResponseEntity<Resource> downloadDocumentById(String caseDocumentId) {
```

Comment:
same here, its not service job to be worried about response entities, thats the controllers job as its the api layer

Reply:
done.
I created DownloadResponse object